doc/source/api.rst

```diff
@@ -646,7 +646,10 @@ strings and apply several methods to it. These can be accessed like
 Categorical
 ~~~~~~~~~~~
 
-If the Series is of dtype ``category``, ``Series.cat`` can be used to change the the categorical
+.. autoclass:: api.types.CategoricalDtype
+   :members: categories, ordered
```

Comment:
yep, we should use the auto* things more readily in other places. Maybe make an issue about this.

Reply:
I don't think we in general need to do this, as for most functions/methods, we already have the generated pages to link to

Reply:
I just did this instead of `autosummary` since there are a bunch of unrelated methods that are just there for NumPy duck-typing.